spec.v2.yaml

```diff
@@ -90,41 +90,54 @@ data:
     # the runtime block so that it matches the stream the module was built
     # against.  Multiple builds result in multiple output modulemd files.
     # See below for an example.
-    # TODO: Provides, conflicts, obsoletes, recommends, etc.
-    #       Do we even need those?
+    # Provides also support stream expansion and indicate what modules names
+    # and streams are provided.  Just like with dependencies, streams can
+    # be inclusive or exclusive lists, with an empty list meaning that any
+    # stream of that module name is provided.
```

Comment:
Could you add some example of why exactly this is desirable? I have some guesses, but I'd really prefer that this was stated explicitly in the specification.

Reply:
Both, please.

From our conversation elsewhere, it seems that this is mainly going to be useful when there are multiple platform branches that should all be treated as Providing the same thing (such as if we decided to do a Fedora 29.1 and 29.2 that should both satisfy Requires: platform:29.